spec/models/passenger_spec.rb

```diff
@@ -9,34 +9,34 @@
 
   describe 'expiration_display' do
     context 'permanent passenger' do
-      it "returns 'None'" do
+      it "returns nil" do
         @passenger.update permanent: true
-        expect(@passenger.expiration_display).to eql 'None'
+        expect(@passenger.expiration_display).to be nil
       end
     end
-    context 'doctors note with expiration date' do
+    context 'temporary passenger' do
       it 'returns the expiration date' do
         date = 14.days.since
         create :doctors_note, passenger: @passenger, expiration_date: date
         expect(@passenger.expiration_display).to eql date.strftime('%m/%d/%Y')
       end
     end
-    context 'doctors note no expiration date' do
-      it "returns 'No Note'" do
-        create :doctors_note, passenger: @passenger
-        expect(@passenger.expiration_display).to eql 'No Note'
-      end
-    end
-    context 'no doctors note' do
-      it "returns 'No Note'" do
-        expect(@passenger.expiration_display).to eql 'No Note'
-      end
-    end
   end
 
   describe 'temporary?' do
     it 'gets the not of permanent' do
-      expect(@passenger.temporary?).to eql true
+      expect(@passenger.temporary?).to be true
```

Comment:
`expect(passenger).to be_temporary`

Reply:
So Rspec can take the boolean columns and test them in this way? That's pretty cool.

Reply:
It's even more generic than that — you can write `be_` followed by anything, and it will call that something as a predicate method (so `be_bananas` calls `bananas?`).